Sources/TuistEnvKit/Installer/Installer.swift

```diff
@@ -77,15 +77,15 @@ final class Installer: Installing {
     }
 
     func install(version: String, temporaryDirectory: AbsolutePath) throws {
-        let bundleURL: URL? = try googleCloudStorageClient.tuistBundleURL(version: version).toBlocking().first() ?? nil
-
-        if let bundleURL = bundleURL {
-            try installFromBundle(
-                bundleURL: bundleURL,
-                version: version,
-                temporaryDirectory: temporaryDirectory
-            )
-        }
+        let release = try githubClient.dispatch(resource: GitHubRelease.release(repositoryFullName: Constants.githubSlug, version: version)).toBlocking().first?.object
```

Comment:
I'm not sure if it's such a good idea to use `toBlocking()` with an artificial timeout of ten seconds (which might be too short if you have slow connection in some cases). But using `sink` is a bit clunky. I think our codebase is ripe for using `async/await` but we'll have to wait a while before making that jump, so I guess it's fine to leave as-is for now.

Reply:
Agree. I think long-term we should cascade async/await up to the CLI command. Because doing now would require changing the interfaces of all the methods to expose publishers, I think it's better if we leave it like this for now. 10 seconds is a lot though or getting a JSON response from an API. If we notice it's not enough we can adjust it.